openshift-hack/check-fmt.sh

```diff
@@ -27,20 +27,18 @@ OPENSHIFT_CI=${OPENSHIFT_CI:-""}
 # script is located.
 cd "$REPO_ROOT"
 
-function runGoimports() {
-    # Goimports acting like gofmt. So, no need to rum fmt separately
-    local GOIMPORTS_PATH=$LOCAL_BINARIES_PATH/goimports
-    go-get-tool "$GOIMPORTS_PATH" golang.org/x/tools/cmd/goimports
-    $GOIMPORTS_PATH -e -w ./cmd/ ./pkg/
-    echo "fmt and goimports done"
+function runGofmt() {
+    for TARGET in "${@}"; do
```

Comment:
So do we need to pass this something? What's the intention here? Doesn't look like it's doing anything where it is called right now

Reply:
Well, I just understood that this commit is a mistake. We generate the code with gofmt anyway because of new changes in golang 1.17. It means we need to use 1.16 for now to prevent the error.